scripts/get

```diff
@@ -157,15 +157,21 @@ if command -v cosign >/dev/null; then
         curl_retry "$BASE_URL/$FILE" -o "$FILE"
     done
 
+    GIT_REF=refs/heads/main
```

Comment:
This will not work for our release branches, right?

Reply:
Good point :-) I'll improve the script to cover this. Let me know if there's more cases that I am not aware of :-)

Reply:
Oh, you're right. I mean, I assume users who care will stick to the tag or commit to be downloaded. So I guess we're fine, thank you for double checking!